static/src/javascripts/__flow__/types/acquisitions.js

```diff
@@ -0,0 +1,5 @@
+declare type AcquisitionsEpicTemplateCopy = {
```

Comment:
I would move `EngagementBannerParams` in here from `ab-tests.js`

Reply:
👍